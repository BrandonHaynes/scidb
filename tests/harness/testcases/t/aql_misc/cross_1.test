--setup
create  array A <a:int32>[x=0:2,3,0]
create  array C <a:int32>[x=0:2,3,0]
store(build(A,x+1),A)
store(build(A,x+10),B)
store(build(C,x+100),C)

--test
--aql update A set a = a + 10

--aql update C set a = a + 50

--aql select * from A cross join B
--aql select * from A cross join C
--aql select * into D from A cross join A

--cleanup
remove(A)
remove(B)
remove(C)
remove(D)
