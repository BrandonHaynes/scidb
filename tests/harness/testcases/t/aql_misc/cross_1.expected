Query was executed successfully

Query was executed successfully

{x} a
{0} 1
{1} 2
{2} 3

{x} a
{0} 10
{1} 11
{2} 12

{x} a
{0} 100
{1} 101
{2} 102

{x} a
{0} 11
{1} 12
{2} 13

{x} a
{0} 150
{1} 151
{2} 152

{x,x} a,a
{0,0} 11,10
{0,1} 11,11
{0,2} 11,12
{1,0} 12,10
{1,1} 12,11
{1,2} 12,12
{2,0} 13,10
{2,1} 13,11
{2,2} 13,12

{x,x} a,a
{0,0} 11,150
{0,1} 11,151
{0,2} 11,152
{1,0} 12,150
{1,1} 12,151
{1,2} 12,152
{2,0} 13,150
{2,1} 13,151
{2,2} 13,152

{x,x_2} a,a_2
{0,0} 11,11
{0,1} 11,12
{0,2} 11,13
{1,0} 12,11
{1,1} 12,12
{1,2} 12,13
{2,0} 13,11
{2,1} 13,12
{2,2} 13,13

Query was executed successfully

Query was executed successfully

Query was executed successfully

Query was executed successfully

